src/app/query/stacks/balance/balance.query.ts

```diff
@@ -15,37 +22,62 @@ const balanceQueryOptions = {
   refetchOnReconnect: false,
 } as const;
 
+function initAmountsAsMoney(balances: AddressBalanceResponse) {
```

Comment:
I created a `balance.utils.ts` file in my btc branch that might be good to move this too later?

Reply:
It does handle data very specific to the query fn though. Isn't it better to keep the logic closer there?